proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host constraints required by the node. These may not be required by a 
+  // vendor, but it can be used to provide requirements for the node.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint is a constraint on the host where the topology is running.
+message HostConstraint{
+  oneof constraint{
+    // Includes the required sysctl settings that need to be configured
+    KernelParam sysctl_constraint = 1;
+  }
+}
+
+// Kernel parameter for Sysctl settings for the host
+message KernelParam{
+  // Actual name of the kernel parameter for sysctl setting
+  string name = 1;
+  // Type of the sysctl setting which could be an array,
```

Comment:
This comment is wrong as currently the only constraint is a BoundedInteger.  I don't think you need a comment for the one of.  The comment for BoundedInteger below is sufficient.

Reply:
Fixed.